deploy/helm/templates/beyla/daemon-set.yaml

```diff
@@ -0,0 +1,110 @@
+{{- if .Values.beyla.enabled }}
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ include "common.fullname" (tuple . "-beyla") }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ include "common.labels" . | indent 4 }}
+  annotations:
+{{ include "common.annotations" . | indent 4 }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ include "common.fullname" (tuple . "-beyla") }}
+  template:
+    metadata:
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/beyla/configmap.yaml") . | sha256sum }}
+{{ include "common.annotations" . | indent 8 }}
+      labels:
+        app: {{ include "common.fullname" (tuple . "-beyla") }}
+{{ include "common.labels" . | nindent 8 }}
+{{ include "common.pod-labels" . | indent 8 }}
+        app.kubernetes.io/component: workload
```

Comment:
What's the purpose of this label?

Reply:
There is no purpose, it was in original resource in beyla's chart, so I left it there